Subject: [PATCH] Make it
---
Index: spksrc.install.mk
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/spksrc.install.mk b/spksrc.install.mk
--- a/spksrc.install.mk	(revision 9ab1cb3cfee6d14c779c3d60cc8c36aa85a81421)
+++ b/spksrc.install.mk	(date 1759264019695)
@@ -49,6 +49,7 @@
 	$(create_target_dir)
 	@mkdir -p $(INSTALL_DIR)/$(INSTALL_PREFIX)
 	find $(INSTALL_DIR)/$(INSTALL_PREFIX)/ \! -type d -printf '%P\n' | sort > $@
+	dos2unix $@
 
 pre_install_target: install_msg_target $(PRE_INSTALL_PLIST)
 
@@ -63,6 +64,7 @@
 $(INSTALL_PLIST):
 	find $(INSTALL_DIR)/$(INSTALL_PREFIX)/ \! -type d -printf '%P\n' | sort | \
 	  diff $(PRE_INSTALL_PLIST) -  | grep '>' | cut -d' ' -f2- > $@
+	dos2unix $@
 
 install_correct_lib_files: $(INSTALL_PLIST)
 	@for pc_file in `grep -e "^lib/pkgconfig/.*\.pc$$" $(INSTALL_PLIST)` ; \
